crates/core/src/db/datastore/system_tables.rs

```diff
@@ -173,26 +173,30 @@ macro_rules! st_fields_enum {
 // WARNING: For a stable schema, don't change the field names and discriminants.
 st_fields_enum!(enum StTableFields {
     "table_id", TableId = 0,
-    "table_name", TableName = 1,
+    "name", TableName = 1,
     "table_type", TableType = 2,
     "table_access", TablesAccess = 3,
 });
```

Comment:
If you're going to rename `table_name` to name, might as well do `table_type` and `table_access`.

Incidentally, I don't know what `table_type` even is. 

Or better yet, I would just leave these names as is to reducer code churn and errors and then we can do a final pass at the end. So leave it as:

```rust
st_fields_enum!(enum StTableFields {
    "table_id", TableId = 0,
    "table_name", TableName = 1,
    "table_name", TableName = 1,
    "table_type", TableType = 2,
    "table_access", TablesAccess = 3,
});
```

Reply:
I agree to keep the prefix for the columns, that helps with debugging.

Reply:
Yeah and joins.